settings.gradle.kts

```
@@ -1,11 +1,18 @@
-rootProject.name = "crowdproj-product-groups"
+rootProject.name = "product-groups"
```

Не стоит сокращать наименования
```suggestion
rootProject.name = "crowdproj-product-groups"
```